src/test/java/eu/tsystems/mms/tic/testerra/plugins/selenoid/ExclusiveSessionSelenoidVideoTest.java

```diff
@@ -40,4 +43,21 @@ public void testT03_FailingTestWillStopVideo() {
         driver.get("https://golem.de");
         Assert.fail("This should fail.");
     }
+
+    @Test(dependsOnMethods = "testT03_FailingTestWillStopVideo", alwaysRun = true)
+    public void test_Video_is_present_after_execution() {
```

Comment:
Just to get it right.
Shouldn't we use the same style of writing for all methods?

This class now contains test methods like 
`testT03_FailingTestWillStopVideo` and `test_Video_is_present_after_execution`.

This is jsut a formal thing, but we should keep it identical torugh the project, at least classes.

Reply:
```suggestion
    public void test_VideoIsPresent_after_FailedTestWithCollectedAssertions() {
```